mods/ts/rules/defaults.yaml

```diff
@@ -721,3 +721,28 @@
 	CustomSelectionSize:
 		CustomBounds: 220,220
 
+^Gate:
+	Inherits: ^Building
+	Valued:
+		Cost: 250
+	Health:
+		HP: 350
+	Armor:
+		Type: Heavy
+	LineBuildNode:
```

Comment:
This doesn't seem to work.  Walls are not visually connecting to the gates like they do for component towers.